src/vector/jitsi/index.ts

```diff
@@ -345,9 +338,13 @@ function mapLanguage(language: string): string {
 // audio input it can find, while an input of null instructs it to start muted,
 // and a non-nullish input specifies the label of a specific device to use.
 // Same for video inputs.
-function joinConference(audioInput?: string | null, videoInput?: string | null): void {
+async function joinConference(audioInput?: string | null, videoInput?: string | null): Promise<void> {
     let jwt;
     if (jitsiAuth === JITSI_OPENIDTOKEN_JWT_AUTH) {
+        // Request credentials, give callback to continue when received
+        openIdToken = await widgetApi.requestOpenIDConnectToken();
```

Comment:
now that `openIdToken` is only used here (and in `createJWTToken`, where we can just pass it in): could we make it a local, rather than a global, variable?